File: Java/TCPClient.java
import java.io.*;

public class TCPClient {
    public static void main(String... args) throws IOException {
        String host = args[0];
        int port = Integer.valueOf(args[1]);
        String toSend = "";
        for (int i = 2; i < args.length; i++) {
            toSend += args[i];
            if (i+1 < args.length)
                toSend += " ";
        }
        System.out.println("Connecting to " + host + " on " + port + " sending " + toSend);

        // Now it's up to you to connect and send and all that
    }
}

File: Python/tcpclient.py
import sys

# Command-line args are stored in the "argv" array in "sys"
# So the first command-line arg is sys.argv[0]
host = sys.argv[1]
port = int(sys.argv[2])
instrs = sys.argv[3:]

print("Opening connection to " + host + " on " + port + " passing " + instrs)

# Now it's up to you to connect and send and all that    